frontend/app/src/components/section-components/chat/channel-header.tsx

```diff
@@ -38,26 +37,25 @@ function ChannelHeader({
             {showForm && (
               <CreateChannelForm
                 setShowForm={setShowForm}
-                setActiveChannelId={setActiveChannelId}
               />
             )}
           </div>
         </div>
       </div>
-      {!channelsData}
-      {channelsData.isLoading && <LoadingSpinner />}
-      {channelsData.isError && (
+      {!channels}
+      {channels.isLoading && <LoadingSpinner />}
+      {channels.isError && (
```

Comment:
could do 
`
      {(channels || channels.isLoading )&& <LoadingSpinner />}
      {(!channels || channels.isError )&& (....`

Reply:
and we remove `{!channels}`

Reply:
in that way we can handle the error case